### DISCUSSION TITLE
Help with compensation

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#676] [General]
Is anyone available to system the two three problems I have in converting my project from BKP5 to BKP6-CoreuiV2?
ref #668 
Not free of course...
I hope someone have a time to help me..
Thanks

**Comment by jcastroa87** [comment]
Hello @blondie63 what kind of problem do you have?

If you send us details maybe we can help you.

Cheers.

**Reply by blondie63** [reply]
@jorgetwgroup You can find details here #668 but let me know if you need more
Thanks

**Comment by blondie63** [comment]
I've found the problem !! Issues about bootstrap include !
